pylint_tweaks

```diff
@@ -0,0 +1,5 @@
+[MASTER]
+load-plugins = edx_lint.pylint
```

Comment:
Shouldn't this file be named `pylintrc_tweaks`?  Its content isn't reflected in pylintrc below